File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/pptnote.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PPTNote:
    """批注"""
    id: str
    content: str
    time: int
    user: str
    el_id: str = None
    replies: List[PPTNoteReply] = None

File: sdkwork-backend-sdk-typescript/src/types/pptnote.ts
import type { PPTNoteReply } from './pptnote-reply';

/** 批注 */
export interface PPTNote {
  /** 批注ID */
  id: string;
  /** 批注内容 */
  content: string;
  /** 批注时间 */
  time: number;
  /** 批注用户 */
  user: string;
  /** 元素ID */
  elId?: string;
  /** 回复列表 */
  replies?: PPTNoteReply[];
}
